src/pykeen/models/unimodal/tucker.py

```diff
@@ -148,9 +148,9 @@ def _scoring_function(
         """
         Evaluate the scoring function.
 
-        Compute scoring function W x_1 h x_2 r x_3 t as in the official implementation, i.e. as
+        Compute scoring function W \times_1 h \times_2 r \times_3 t as in the official implementation, i.e. as
 
-            DO(BN(DO(BN(h)) x_1 DO(W x_2 r))) x_3 t
+            DO(BN(DO(BN(h)) \times_1 DO(W \times_2 r))) \times_3 t
 
         where BN denotes BatchNorm and DO denotes Dropout
```

Comment:
maybe we can drop this part of the docstring, since the same text is already in the class docstring.